domain-bie-events/svc-claim-tracker/src/app/api.py

```diff
@@ -0,0 +1,100 @@
+from contextlib import asynccontextmanager
+from typing import Any, AsyncIterator
+
+import uvicorn
+from fastapi import Depends, FastAPI, Request, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+from sqlalchemy.exc import SQLAlchemyError
+from sqlmodel import Session
+
+from app.database.config import get_session, init_db
+from app.database.tracked_claim_repo import tracked_claim_repo
+from app.model.healthcheck import HealthErrorResponse, HealthResponse
+from app.model.tracked_claim import TrackedClaim
+from app.schema.track_claim import TrackClaimRequest, TrackClaimResponse
+from app.util.logger import logger
+
+CONNECT_TO_DATABASE_FAILURE = 'Cannot connect to database.'
+
+
+@asynccontextmanager
+async def lifespan(api: FastAPI) -> AsyncIterator[None]:
+    init_db()
+    yield
+
+
+app = FastAPI(
+    title='Claim Tracker for BIE Claim Events',
+    description='This API accepts requests to track a claim lifecycle by BIE Claim Events.',
+    contact={},
+    version='v0.1',
+    license={'name': 'CCO 1.0', 'url': 'https://github.com/department-of-veterans-affairs/abd-vro/blob/master/LICENSE.md'},
```

Comment:
```suggestion
    license={'name': 'CCO 1.0', 'url': 'https://github.com/department-of-veterans-affairs/abd-vro/blob/main/LICENSE.md'},
```
(branch was renamed to `main`)